package-lock.json

```diff
@@ -15664,8 +15656,7 @@
       "integrity": "sha512-xiqMQR4xAeHTuB9uWm+fFRcIOgKBMiOBP+eXiyT7jsgVCq1bkVygt00oASowB7EdtpOHaaPgKt812P9ab+DDKA==",
       "extraneous": true,
       "os": [
-        "darwin",
-        "linux"
+        "darwin"
```

Comment:
this is a bit tricky but you need to add "linux" to the list here to allow github action to install the dependency properly on ubuntu machine.